### PR TITLE
fix for bad PR #1584

### PR DESCRIPTION
**Pull request by tardyp**
Signed-off-by: Pierre Tardy <pierre.tardy@intel.com>

### CODE REVIEW

**Comment by yetanotherion** [comment]
:+1:

File: master/buildbot/data/base.py [hunk 1]
```diff
@@ -88,6 +88,9 @@ def control(self, action, args, kwargs):
     def startConsuming(self, callback, options, kwargs):
         raise NotImplementedError
 
+    def __repr__(self):
+        return "endpoint for" + self.pathPatterns
```
Comment (sa2ajj) [inline master/buildbot/data/base.py]: don't we try to avoid string summation?  and use interpolation instead?

[nit] missing space

Comment (djmitche) [inline master/buildbot/data/base.py]: I don't have a preference of one approach over the other, but yes -- the missing space needs to be added.  With that fixed, I think this is ready to merge.

**Comment by djmitche** [comment]
I'll add the space while merging..

**Pull request merged**